docs/includes/install-cli.md

```diff
@@ -7,6 +7,13 @@
 1. Each download is the `nuget.exe` file directly. Instruct your browser to save the file to a folder of your choice. The file is *not* an installer; you won't see anything if you run it directly from the browser.
 1. Add the folder where you placed `nuget.exe` to your PATH environment variable to use the CLI tool from anywhere.
 
+    - Open the CMD
+    - Run the following command:
```

Comment:
I don't think this should be added. 

It's misusing a totally different feature unrelated to this.

Reply:
Do you want to abandon this PR? Or there is something you want to be updated?

Reply:
I agree with @nkolev92 on this. The dotnet tools folder is managed by the dotnet cli, so the NuGet team should not encourage people to manually add/remove executables there.

If you'd like the docs to say that you can copy nuget.exe to a folder that's already on the path, then I'd be happy to approve the PR if you change it to say that. But I don't support using examples that encourage putting nuget.exe in other application's directories.

I appreciate that you took the time to write and submit this, but I don't think it should be included in the official docs as-is.

Reply:
@zivkan Yes, you are right. Would you like to add a reference(the exact command) to [setx](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/setx)? Or you just want to let people know that they have the option to move the `nuget.exe` to a folder that's already on the env path?

Reply:
I think examples of how to use the product being documented is important, but I don't feel the same way about documenting other product/features. How to set environment variables is out of scope for NuGet, so I'd prefer to link to other docs that explain how to do that, but I'm unable to find docs on that on docs.microsoft.com.

Reply:
It seems to me that you don't like any change to the current version of the document. Consequently, I am closing this PR.